services/asset/nodes/parsers.go

```diff
@@ -33,13 +37,9 @@ func (node *Node) toMapRecursive(level int) map[string]any {
 		}
 		nodeMap["nodes"] = nodes
 	}
-	if len(node.NoradId) != 0 {
+	if node.Kind == "SATELLITE" {
```

Comment:
This block (line 40 to 49) looks useless but does it work because of the PostRead or can it be removed ?

Reply:
It's not useless cause caues it come after PostRead